Framework/Algorithms/src/GetEiMonDet2.cpp

```diff
@@ -38,11 +39,11 @@ const static std::string FIT_STATUS_SUCCESS("success");
  */
 namespace IndexTypes {
 /// Tag for detector ids
-const static std::string DETECTOR_ID("DetectorID");
+const static std::string DETECTOR_ID("Detector ID");
```

Comment:
This version of GetEiMonDet was added to Mantid quite recently, and the only user at the moment would be me while I'm developing the `DirectILLReduction` algorithm. So, I think it is still OK to make such API breaking changes. The last chance, actually.